core/src/main/java/com/klaytn/caver/rpc/Klay.java

```diff
@@ -0,0 +1,1295 @@
+     * It returns false if the account is an EOA or a smart contract account which doesn't have codeHash.
+     * It sets block tag to "LATEST".
+     * @param address The account address
+     * @return Boolean
+     */
+    public Request<?, Boolean> isContractAccount(String address) {
+        return isContractAccount(address, DefaultBlockParameterName.LATEST);
+    }
+
+    /**
+     * Returns true if an input account has a non-empty codeHash at the time of a specific block number.
+     * It returns false if the account is an EOA or a smart contract account which doesn't have codeHash.
+     * @param address The account address
+     * @param blockNumber The block number..
+     * @return Boolean
+     */
+    public Request<?, Boolean> isContractAccount(String address, long blockNumber) {
+        DefaultBlockParameterNumber blockParameterNumber = new DefaultBlockParameterNumber(blockNumber);
+
+        return isContractAccount(address, blockParameterNumber);
+    }
+
+    /**
+     * Returns true if an input account has a non-empty codeHash at the time of a specific block number.
+     * It returns false if the account is an EOA or a smart contract account which doesn't have codeHash.
+     * @param address The account address
+     * @param blockTag The string "latest", "earliest" or "pending"
+     * @return Boolean
+     */
+    public Request<?, Boolean> isContractAccount(String address, DefaultBlockParameter blockTag) {
+        return new Request<>(
+                "klay_isContractAccount",
+                Arrays.asList(
+                        address,
+                        blockTag.getValue()
+                ),
+                web3jService,
+                Boolean.class);
+    }
+
+    /**
+     * The sign method calculates a Klaytn-specific signature.
+     * NOTE : The address to sign with must be unlocked.
+     * @param address The account address
+     * @param message The message to sign.
+     * @return Bytes
+     */
+    public Request<?, Bytes> sign(String address, String message) {
+        return new Request<>(
+                "klay_sign",
+                Arrays.asList(
+                        address,
+                        message
+                ),
+                web3jService,
+                Bytes.class);
+    }
+
+    /**
+     * Returns the number of most recent block.
+     * @return Quantity
+     */
+    public Request<?, Quantity> getBlockNumber() {
+        return new Request<>(
+                "klay_blockNumber",
+                Collections.<String>emptyList(),
+                web3jService,
+                Quantity.class);
+    }
+
+    /**
+     * Returns information about a block by block number.
+     * It set "isFullTransaction" param to true.
+     * @param blockNumber The block number.
+     * @return KlayBlock
+     */
+    public Request<?, KlayBlock> getBlockByNumber(long blockNumber) {
```

Comment:
KlayBlock use `KlayTransaction.Transaction` for transactions.
How about creating `Block` class to use `Transaction` for transactions?

Reply:
I added Block class